examples/shaders/shaders_mesh_instancing.c

```diff
@@ -61,7 +61,7 @@ int main(void)
     {
         Matrix translation = MatrixTranslate((float)GetRandomValue(-50, 50), (float)GetRandomValue(-50, 50), (float)GetRandomValue(-50, 50));
         Vector3 axis = Vector3Normalize((Vector3){ (float)GetRandomValue(0, 360), (float)GetRandomValue(0, 360), (float)GetRandomValue(0, 360) });
-        float angle = (float)GetRandomValue(0, 10)*DEG2RAD;
+        float angle = (float)GetRandomValue(0, 180) * DEG2RAD;
```

Comment:
Please maintain the [raylib coding conventions](https://github.com/raysan5/raylib/wiki/raylib-coding-conventions).